dbscripts/xml/upgrade.xml

```diff
@@ -148,6 +148,7 @@
 		<migration class="APP\migration\upgrade\v3_4_0\I8151_ExtendSettingValues"/>
 		<migration class="PKP\migration\upgrade\v3_4_0\I8151_ExtendSettingValues"/>
 		<migration class="PKP\migration\upgrade\v3_4_0\I7366_UpdateUserAPIKeySettings"/>
+		<migration class="PKP\migration\upgrade\v3_4_0\I7725_DecisionConstantsUpdate"/>
```

Comment:
This is failing the upgrade tests. I think it should be `APP\migration\...`. 

https://app.travis-ci.com/github/pkp/omp/builds/254968701